### PR TITLE
added php file for upskill form

### PR DESCRIPTION
**Pull request by 81jumpstr**

### CODE REVIEW

File: page-upskill-form.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?php get_header(); ?>
+
+<form>
+  <h1>1UP</h1>
+  <ul class="1up-details">
+    <li>
+	    <label for="bizprojectname">Business/Project Name:</label>
+	    <input type="text" name="bizprojectname">
+	  </li>
```
Comment (royscheeren) [inline page-upskill-form.php]: Indentation, indentation, indentation

File: page-upskill-form.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?php get_header(); ?>
+
+<form>
+  <h1>1UP</h1>
+  <ul class="1up-details">
```
Comment (royscheeren) [inline page-upskill-form.php]: I'm not sure if classes that start with numbers are allowed.

File: page-upskill-form.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	  <input type="radio" name="description" value="project"> Project (for profit)
+	  <input type="radio" name="description" value="business"> Business (fundraiser)
+	  <input type="radio" name="description" value="business"> Business (for profit)
+  </div>
+  
+  <input class="1up-submit" type="submit" value="Submit">
+
+</form>
+
+<?php get_footer(); ?>
```
Comment (royscheeren) [inline page-upskill-form.php:48]: All the indentations

**Review by royscheeren — CHANGES_REQUESTED** [review]

**Comment by 81jumpstr** [comment]
https://github.com/81jumpstr/Setup01/pull/30/commits/f25b525a0b7f8705e7614a643385bada8fd34a84

**Review by royscheeren — APPROVED** [review]

**Pull request merged**